### PR TITLE
Tiny fix readme

### PR DESCRIPTION
**Pull request by yichen88**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [X] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ This project and everyone participating in it is governed by the [PowSyBl Code o
...
+The PowSyBl Core project is not a standalone project. Read on to learn how to modify the core code, be it for fun, for diagnosing bugs, for improving your understanding of the framework, or for preparing pull requests to suggest improvements! PowSyBl Core provides library code to build all kinds of applications for power systems: a complete and extendable grid model, support for common exchange formats, APIs for power simulations an analysis, and support for local or distributed computations. For deployment, powsybl-core also provides an iTools, a tool to build cross-platform integrated command line applications. To build cross-platform graphical applications, please visit the PowSyBl GSE repository https://github.com/powsybl/powsybl-gse page.
```
Comment (yichen88) [inline README.md]: PowSyBl <-- PowSyBle

Comment (yichen88) [reply README.md]: kinds <-- kind
an iTools <-- a iTools

**Comment by yichen88** [comment]
typo fix:
kinds<-kind
an iTools<-a iTools
command-line<-command line
PowSybl<-Powsyble

**Comment by jonenst** [comment]
Thanks, I added other minor changes on top of your PR

**Pull request merged**